cpp/benchmarks/CMakeLists.txt

```diff
@@ -14,9 +14,44 @@
 # limitations under the License.
 #=============================================================================
 
-add_library(cudf_datagen STATIC
-                   "${CMAKE_SOURCE_DIR}/benchmarks/common/generate_benchmark_input.cpp"
-)
+cmake_minimum_required(VERSION 3.14...3.18 FATAL_ERROR)
+
+project(CUDF_BENCHMARKS VERSION 0.18.0 LANGUAGES C CXX CUDA)
+
+# Find or install GoogleTest
+CPMFindPackage(NAME GTest
```

Comment:
gtest shouldn't be needed by gbench if testing is off.

Reply:
the `cudf_datagen` and benchmark targets link against `gtest`, I think because `generate_benchmark_input.cpp` includes headers from `cudf_test`.